rpcs3/rpcs3qt/pad_motion_settings_dialog.cpp

```diff
@@ -171,9 +171,11 @@ pad_motion_settings_dialog::pad_motion_settings_dialog(QDialog* parent, std::sha
 	// Use thread to get button input
 	m_input_thread = std::make_unique<named_thread<std::function<void()>>>("UI Pad Motion Thread", [this]()
 	{
+		u64 sleep_until = get_system_time();
+
 		while (thread_ctrl::state() != thread_state::aborting)
 		{
-			thread_ctrl::wait_for(1000);
+			thread_ctrl::wait_until(&sleep_until, 1000);
```

Comment:
Please don't make this even faster and more complex It's bad enough that it's this fast